src/states/endsurvival.ts

```diff
@@ -1,20 +1,29 @@
 import 'phaser-ce';
 import MenuState from './menuState';
+import {Audio} from '../assets';
 
 export default class EndSurvival extends MenuState {
 
+    waves;
+
     constructor() {
         super('You have survived 0 waves!');
     }
 
     public init(waves) {
+        this.waves = waves;
         this.setTitle('You have survived ' + waves + ' waves!');
     }
 
     public create(): void {
         super.create();
         this.addOption('Play again', 'survival', null, this.game.world.centerY - 100);
         this.addOption('Return to main menu', 'mainScreen', null, this.game.world.centerY);
+        if (this.waves === 0) {
+            this.game.add.sound(Audio.AudioFemaleWrong.getName()).play();
```

Comment:
maybe mission failed audio is a better fit here (possibly paired with a mission completed at the end of 3 waves) -> this is open to discussion

Reply:
Done